### PR TITLE
My data

### PR DESCRIPTION
**Pull request by ivoytov**
latest coefficients and first Bay Area model. The +garage,619.36856072660555 line is worrisome as it's clearly too high. 90% of the dataset is marked as not having a garage, which is impossible. I suspect only the nicest properties are labeled with parking so it's capturing some of that independent variable in this regression.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,372 +0,0 @@
-GIT
```
Comment (garyjob) [inline Gemfile.lock]: Do not delete this file from your code base.

File: lib/tasks/model_files/model_covs_bay_area_20151101.csv [hunk 2]
```diff
@@ -0,0 +1,27 @@
+,bedrooms,bathrooms,level,garage,age,elevation,neighborhood[ALAMEDA]:sqft,neighborhood[ALBANY]:sqft,neighborhood[BAY AND OCEAN]:sqft,neighborhood[BERKELEY]:sqft,neighborhood[CASTRO VALLEY]:sqft,neighborhood[DUBLIN]:sqft,neighborhood[EMERYVILLE]:sqft,neighborhood[FREMONT]:sqft,neighborhood[HAYWARD]:sqft,neighborhood[LIVERMORE]:sqft,neighborhood[MILPITAS]:sqft,neighborhood[NEWARK]:sqft,neighborhood[OAKLAND]:sqft,neighborhood[PACIFICA]:sqft,neighborhood[PLEASANTON]:sqft,neighborhood[SAN LEANDRO]:sqft,neighborhood[SAN LORENZO]:sqft,neighborhood[SOUTH SAN FRANCISCO]:sqft,neighborhood[UNION CITY]:sqft,neighborhood[WALNUT CREEK]:sqft
```
Comment (garyjob) [inline lib/tasks/model_files/model_covs_bay_area_20151101.csv]: What is this file for?

File: lib/tasks/model_files/model_covs_ph_20151101.csv [hunk 3]
```diff
@@ -0,0 +1,65 @@
...is Park]:sqft,neighborhood[Kyrene-Superstition]:sqft,neighborhood[Laveen]:sqft,neighborhood[Lindon Park]:sqft,neighborhood[Mach 8]:sqft,neighborhood[Maple-Ash]:sqft,neighborhood[Maryvale]:sqft,neighborhood[McClintock]:sqft,neighborhood[Meyer Park]:sqft,neighborhood[Mitchell Park West]:sqft,neighborhood[New Village]:sqft,neighborhood[North Gateway]:sqft,neighborhood[North Mountain]:sqft,neighborhood[North Scottsdale]:sqft,neighborhood[Northeast 0]:sqft,neighborhood[Northeast 3]:sqft,neighborhood[Northeast 4]:sqft,neighborhood[Northwest]:sqft,neighborhood[Ntna-College]:sqft,neighborhood[Ntna-Indian Bend]:sqft,neighborhood[Ntna-West Rio]:sqft,neighborhood[Optimist Park Ne]:sqft,neighborhood[Optimist Park Sw]:sqft,neighborhood[Paradise Valley]:sqft,neighborhood[Pepperwood]:sqft,neighborhood[Pinnacle Peak]:sqft,neighborhood[Riverside]:sqft,neighborhood[Rural-Geneva]:sqft,neighborhood[Shalimar]:sqft,neighborhood[South Mountain]:sqft,neighborhood[South Scottsdale]:sqft,neighborhood[Southeast 0]:sqft,neighborhood[Southwest]:sqft,neighborhood[Sunburst Farms]:sqft,neighborhood[Sunset]:sqft,neighborhood[Tempe Gardens]:sqft,neighborhood[Tempe Royal Estates]:sqft,neighborhood[West Central]:sqft
```
Comment (garyjob) [inline lib/tasks/model_files/model_covs_ph_20151101.csv]: What is this file for?

Comment (ivoytov) [inline lib/tasks/model_files/model_covs_ph_20151101.csv]: It's the covariance matrix. It's needed to compute an estimated range for
each property, which is a feature we need to add so we can filter out
predictions where we don't have enough data to make a valid estimate.

**Pull request closed without merging**